sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -230,6 +230,7 @@ CREATE TABLE `form_eye_acuity` (
   `LIOSVA`        varchar(25) NOT NULL,
   `WODVANEAR`     varchar(25)  DEFAULT NULL,
   `OSVANEARCC`    varchar(25)  DEFAULT NULL,
+  `BINOCVA`       varchar(25)  DEFAULT NULL,
```

Comment:
Would add this at the bottom of the upgrade script with it's own If block (otherwise folks that are running from most recent codebase won't get this change)

Reply:
above comment not yet addressed